### DISCUSSION TITLE
Vulkan support as soon as it's available in MG ?

### DISCUSSION THREAD

**Discussion by orosbogdan** [kniEngine/kni#1836] [General]
I want to start my new project using KNI, but I want to know if there is any commitment for syncing with MG for the vulkan backend once it's available over there.

**Comment by nkast** [comment]
The two libraries have a different architecture.
It makes more sense to implement it from the ground up, following the official tutorials and samples, that to try and backport the MG implementation. Other options are to write a backend against FNA3D, the FNA's native Api that implements GL/Vulkan and DX12, or SDL_GPU.
